src/bench/murmur_hash.cpp

```diff
@@ -8,7 +8,7 @@
 static void Murmur3(benchmark::State &state)
 {
     std::vector<uint8_t> in(32, 0);
-    unsigned x;
+    unsigned x = 0;
     while (state.KeepRunning())
```

Comment:
nit: also unsigned int?

Reply:
you are correct, the policy is to use `unsigned int` or `uint32_t`

Reply:
fixed and pushed.